app/src/App.js

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+
+import * as apiClient from "./apiClient";
+
+const App = () => {
+  const [tasks, setTasks] = React.useState([]);
+
+  const today = new Date();
+  const [date, setDate] = React.useState(today);
+  const loadTasks = async () => setTasks(await apiClient.getTasks());
```

Comment:
apiClient.getTasks could be modified to accept a date since your journal entries/tasks will be linked to a date.